client/app.js.jsx

```diff
@@ -23,7 +28,7 @@ render(
     <Provider store={store}>
       <Router
         history={history}
-        render={applyRouterMiddleware(useScroll(), useRelativeLinks())}
+        render={applyRouterMiddleware(useRelativeLinks(), useScroll() )}
```

Comment:
does scroll behavior work as expected here?  i remember something about one breaking the other and visa versa